plugins/generator-1.16.4/forge-1.16.4/procedures/entity_send_chat.java.ftl

```diff
@@ -1,3 +1,3 @@
-if(${input$entity} instanceof PlayerEntity && !${input$entity}.world.isRemote()){

```

Comment:
How does removal of () fix this technically speaking?

Reply:
I admit I don't know, but it does fix the problem.

Reply:
Please print value of both variants before the if to see if they are different as this may be important for other cases too

Reply:
isRemote is a public variable of World, so the class World surely doesn't have a method isRemote().

Reply:
Since 1.16.x IWorld has it to replace field access which is not the best practice.

Reply:
> Please (just for test) print value of both variants before the if to see if they are different as this may be important for other cases too

I tested, the way I made it is the only way for it to work.

Reply:
Actually, it works fine with `isRemote()`